tests/Controller/SecurityTest.php

```diff
@@ -118,4 +118,17 @@ public function testUpdateUser()
         $this->assertInputValueSame('account[name]', $newName);
     }
     
+
+    public function testGetForgotPwdForm()
+    {
+        $crawler = $this->client->request('GET', '/security/lost_pwd');
+        $this->assertResponseIsSuccessful();
+    }
+    /**
+     * todo:
```

Comment:
?

Reply:
forgot to delete comments. fixed in lost-password branch